roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/UserManagement.java

```diff
@@ -323,6 +321,10 @@ public static void removeGroup(RodaUser user, String groupname)
     registerAction(user, "UserManagement", "removeGroup", null, duration, "groupname", groupname);
   }
 
+  // TODO: Methods bellow this line should also checkRoles? If so, a RodaUser is needed.
```

Comment:
@luis100 @hsilva-keep The methods bellow this comment, are all related with user registration/recover and don't have access to the RodaUser that is calling the method. To call chechRoles(...) the RodaUser must come from somewhere. Please advise on what to do.